plugins/inputs/opcua/opcua_client.go

```diff
@@ -180,6 +187,11 @@ const sampleConfig = `
   #  {name="", namespace="", identifier_type="", identifier=""},
   #  {name="", namespace="", identifier_type="", identifier=""},
   #]
+
+  ## Enable workarounds required by some devices to work correctly
+  # [inputs.opcua.workarounds]
+  ## Set valid status codes
+  # valid_status_codes = ["0x0", "0xC0"]
```

Comment:
Additional to the comment of @reimda (also applying here), I would appreciate if you increase the indentation level of this section.
```suggestion
    ## Set valid status codes
    # valid_status_codes = ["0x0"]
```
This should also be done in the README to keep both in sync.